src/instructlab/train/linux_train.py

```diff
@@ -169,15 +168,21 @@ def linux_train(
     try:
         device = torch.device(train_device)
     except RuntimeError as e:
-        ctx.fail(str(e))
+        click.secho(
+            str(e),
+            fg="red",
+        )
+        raise click.exceptions.Exit(1)
```

Comment:
i think we should replace click references in any code that is not in the `instructlab/cli` package

Reply:
true, forgot this file since no other changes were necessary